### PR TITLE
Update VoiceAndTone.stories.mdx

### PR DESCRIPTION
**Pull request by americanrefugee**
Changes to text formatting for "Examples"

### CODE REVIEW

**Comment by americanrefugee** [comment]
@gsouquet The checks failed, I don't know what I did wrong :(

File: stories/content/VoiceAndTone.stories.mdx [hunk 1]
```diff
@@ -87,14 +87,17 @@ In the same way that 80% of communication between humans is non-verbal, the same
       <img src="/img/example_homepage-illustration.jpg" alt="" />
     </td>
     <td>
-      <strong>Illustration on homepage</strong>A simple illustrations shows that
+      <strong>Illustration on homepage</strong> 
+      <p>
```
Comment (gsouquet) [inline stories/content/VoiceAndTone.stories.mdx:91]:
```suggestion
```

File: stories/content/VoiceAndTone.stories.mdx [hunk 2]
```diff
@@ -87,14 +87,17 @@ In the same way that 80% of communication between humans is non-verbal, the same
...
+      A simple illustrations shows that
       enterprise customers can use Element to communicate with outside partners,
       no matter what platform they use.
     </td>
   </tr>
   <tr>
     <td>
       <strong>Verification request</strong>
+      <p>
```
Comment (gsouquet) [inline stories/content/VoiceAndTone.stories.mdx:100]:
```suggestion
```

File: stories/content/VoiceAndTone.stories.mdx [hunk 3]
```diff
@@ -109,6 +112,7 @@ In the same way that 80% of communication between humans is non-verbal, the same
     </td>
     <td>
       <strong>Scanning a QR code</strong>
+      <p>
```
Comment (gsouquet) [inline stories/content/VoiceAndTone.stories.mdx:115]:
```suggestion
```

File: stories/content/VoiceAndTone.stories.mdx [hunk 4]
```diff
@@ -117,6 +121,7 @@ In the same way that 80% of communication between humans is non-verbal, the same
   <tr>
     <td>
       <strong>Device verification successful</strong>
+      <p>
```
Comment (gsouquet) [inline stories/content/VoiceAndTone.stories.mdx:124]:
```suggestion
```

File: stories/content/VoiceAndTone.stories.mdx [hunk 5]
```diff
@@ -130,6 +135,7 @@ In the same way that 80% of communication between humans is non-verbal, the same
     </td>
     <td>
       <strong>No internet connection banner</strong>
+      <p>
```
Comment (gsouquet) [inline stories/content/VoiceAndTone.stories.mdx:138]:
```suggestion
```

File: stories/content/VoiceAndTone.stories.mdx [hunk 6]
```diff
@@ -138,6 +144,7 @@ In the same way that 80% of communication between humans is non-verbal, the same
   <tr>
     <td>
       <strong>Sign out of Element Admin Console</strong>
+      <p>
```
Comment (gsouquet) [inline stories/content/VoiceAndTone.stories.mdx:147]:
```suggestion
```

File: stories/content/VoiceAndTone.stories.mdx [hunk 7]
```diff
@@ -150,7 +157,9 @@ In the same way that 80% of communication between humans is non-verbal, the same
       <img src="/img/example_new-messages.jpg" alt="" />
     </td>
     <td>
-      <strong>New message</strong>A brief sound is used to indicated a new
+      <strong>New message</strong>
+      <p>
```
Comment (gsouquet) [inline stories/content/VoiceAndTone.stories.mdx:161]:
```suggestion
```

**Review by gsouquet — CHANGES_REQUESTED** [review]
The HTML generated by the plugin you're using is incorrect. I've made the updates needed

**Review by gsouquet — APPROVED** [review]
Looks good 👍

**Pull request merged**